components/resolved_ts/src/endpoint.rs

```diff
@@ -378,6 +375,7 @@ pub struct Endpoint<T, E: KvEngine, S> {
     region_read_progress: RegionReadProgressRegistry,
     regions: HashMap<u64, ObserveRegion>,
     scanner_pool: ScannerPool<T, E>,
+    scan_concurrency_semaphore: Arc<Semaphore>,
```

Comment:
Should it support dynamic config change? The existing handle_change_config could be extended to support dynamic config.

Reply:
> Should it support dynamic config change?

Yes, it should supports.

> The existing handle_change_config could be extended to support dynamic config.

Added.